setup.py

```diff
@@ -32,7 +32,7 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-ooyala-player',
-    version='4.0.5',
+    version='4.0.6',
```

Comment:
if we are removing ooyala support completely then it should be a major version bump

Reply:
Ooyala support was mainly removed in #111 where major version bump was made because it was a  backward incompatible change. This one does not break anything or introduce new feature but just do cleanup of already unused code